src/data.js

```diff
@@ -3588,6 +3588,27 @@ module.exports = [
     phone: 'android',
     tags: ['JavaScript', 'TypeScript', 'React', 'Node', 'GraphQl', 'Svelte'],
   },
+  {
+    name: 'Ronie Meque',
+    description:
+      'Web Developer. Loves to help people and hopes to one day actually finish a side project or travel the world, whichever comes firts',
+    url: 'https://ronie.dev/uses',
```

Comment:
Hmm this doesn't look like a uses page to me, would it be cool if we hold off until you create one?

Reply:
You're right, sorry! I will finish it and notify here 😊